### PR TITLE
Update to add new team members!

### PR DESCRIPTION
**Pull request by abbyfischler**
@christinaasquith

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/team**](https://site-54nt3j102.hackclub.dev/team) |
|:-:|
|<a href="https://site-54nt3j102.hackclub.dev/team">Screenshot of /team</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ccB3PeK4ndFfqohq6Ol8t2UNsq9wj2kh-8_ru2mzhTKHIe7dP0S4b06ZplRcfcTp8lVL4ruh879DHDPUE9HxWw.png">(view full size)</a> |







Commit f98358733a07a821cd08e2f41395dc0172a8e545 (https://site-54nt3j102.hackclub.dev).

File: pages/team.js [hunk 1]
```diff
@@ -112,6 +112,34 @@ export default function Team() {
...
+                name="Thea Baren"
+                teamRole="Executive Assistant to the COO"
+                text="Thea will now serve as Christina’s executive assistant."
+                img="/team/thea.png"
+                pronouns="she/her"
+              />
+              <Bio
+                name="Bence"
+                teamRole="Bookkeeper"
+                text="Bence is responsible for  keeping accurate financial books for hq and every org on hack club bank (40,000+ transactions in the last year!)."
```
Comment (cjdenio) [inline pages/team.js]:
```suggestion
                text="Bence is responsible for keeping accurate financial books for HQ and every org on Hack Club Bank (40,000+ transactions in the last year!)."
```

File: pages/team.js [hunk 2]
```diff
@@ -112,6 +112,34 @@ export default function Team() {
...
+                img="/team/bence.png"
+                pronouns="he/him"
+              />
+               <Bio
+                name="Chris Walker"
+                teamRole="Engineer"
+                text="Chris Walker is joining us part-time from San Francisco to ship sinerider and bring mayhem and magic to #gamedev and beyond."
```
Comment (cjdenio) [inline pages/team.js]: Ideally "siderider" would be linked to a page explaining what it is

File: pages/team.js [hunk 3]
```diff
@@ -245,6 +273,13 @@ and participating in academic competitions with his friends.`}
                 img="https://scrapbook.hackclub.com/hugo.y.hu935.png"
                 pronouns="he/him"
               />
+              <Bio
+                name="Ishan Goel"
+                teamRole="Summer Intern"
+                text="Ishan is a 16 year old summer intern from Seattle! This summer he is working on shipping projects with partners to get the word out about hack club, and bring more people into the community."
```
Comment (cjdenio) [inline pages/team.js]:
```suggestion
                text="Ishan is a 16-year-old summer intern from Seattle! This summer he is working on shipping projects with partners to get the word out about Hack Club, and bring more people into the community."
```

**Review by cjdenio — COMMENTED** [review]
Just a few small comments/edits.

File: pages/team.js (hunk 2 above)
Comment (abbyfischler) [reply pages/team.js]: Is there a website to siderider? @cjdenio

Comment (cjdenio) [reply pages/team.js]: I think https://github.com/hackclub/sinerider would be good

Comment (exu3) [reply pages/team.js]: Chris is in the Acknowledgements section already; maybe we move that card here and add the new copy?

**Pull request merged**